dask/base.py

```diff
@@ -630,25 +630,11 @@ def visualize(*args, **kwargs):
 
     filename = kwargs.pop("filename", "mydask")
     optimize_graph = kwargs.pop("optimize_graph", False)
+    traverse = kwargs.pop("traverse", True)
```

Comment:
Can you add `traverse` to the docstring above? While you are at it you could probably also copy some of the docstring text from `compute.args` to make that section more explicit.

Reply:
Good point! Done!